### PR TITLE
Created structure for our webApp with routes and other things

### PR DESCRIPTION
**Pull request by AdityaShekharTiwary**
@author 
Aditya Shekhar Tiwary

Description :- 

- Created Navbar , Home , Upload , About Screens 
- Routes for single page application
- stylings

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+} from "react-router-dom";
+
+
+function App() {
+  return (
+    <Router>
+      <div className="">
+        <Navbar />
+        <Routes>
+          <Route exact path="/home" element={<><Home /></>}></Route>
```
Comment (cookieg13) [inline src/App.jsx:19]: you can remove these extra <> </>

Comment (cookieg13) [reply src/App.jsx:19]: Everything else looks great @AdityaShekharTiwary  , can you add your review too? @Supriya-roy9  we can merge it then

Comment (dgeee13) [inline src/App.jsx:20]: you can remove these extra <> </>

**Review by dgeee13 — CHANGES_REQUESTED** [review]

File: src/components/style.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
+.navbar{
```
Comment (dgeee13) [inline src/components/style.css:1]: Will convert this to tailwind later

**Comment by dgeee13** [comment]
Everything else looks great @AdityaShekharTiwary , can you @Supriya-roy9  add your review too? we can then merge it

**Review by dgeee13 — APPROVED** [review]

**Pull request merged**